poetry.lock

```diff
@@ -387,9 +387,7 @@ version = "0.9.14"
 description = "Python API for communicating with the CARLA server."
 optional = false
 python-versions = "*"
-files = [
```

Comment:
is there a standard way to generate the openpilot lockfile? do we include --dev?

Reply:
`poetry lock --no-update` for stuff like this

Reply:
ok, thats what I did. not sure how carla was added, probably someone did --dev